config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  adapter: async
+
+test:
+  adapter: test
+
+production:
+  adapter: redis
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: fromthepage_production
```

Comment:
Are we sure we're using redis?  Where or how is it configured?

Reply:
@saracarl usually running `redis-server` is enough, but I think this can be safely ignored until/unless we start using Action Cable.